### ISSUE TITLE
Write some user guides

### ISSUE THREAD

**Issue by keisen** [keisen/tf-keras-vis#23]
* Visualizing multiple attention or activation images at once utilizing batch-system of model
* Define various loss functions
* Visualizing attentions with multiple inputs models
* Visualizing attentions with multiple outputs models
* Advanced loss functions
* Tuning Activation Maximization
* Visualizing attentions for N-dim image inputs
